src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,94 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileReader;
+
+public class Storage {
+    private File file;
+
+    /**
+     *
+     * @param file text file where tasks are written into
+     */
+    public Storage(File file) {
+        this.file = file;
+    }
+
+    /**
+     *
+     * @param tasks tasks from taskList which are written into the text file
+     * @throws IOException
+     */
+    public void writeToFile(TaskList tasks) throws IOException {
+       //file = new File("src/main/java/duke.txt");
+
+        FileWriter fw = new FileWriter(file);
+
+        for (Task task: tasks) {
+            fw.write(task.toString() + "\n");
+        }
+        System.out.println("Auto-saved.");
+        fw.close();
+    }
+
+    /**
+     *
+     * @return tasks from text file are loaded and stored into the TaskList upon restarting the bot
+     * @throws IOException
+     */
+    public TaskList loadFile() throws IOException {
+        //file = new File("src/main/java/duke.txt");
+
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+
+        BufferedReader reader = new BufferedReader(new FileReader("src/main/java/duke.txt"));
+        String str;
+        TaskList loadedList = new TaskList();
+
+        while((str = reader.readLine()) != null) {
+
+            if(str.charAt(1) == 'T') {
```

Comment:
Would it be better if you computed `str.charAt(1)` before the if-else block? Would it be simpler if you implemented a switch case instead?